### DISCUSSION TITLE
Just a little Thank you

### DISCUSSION THREAD

**Discussion by Ketchoupii** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#108] [General]
Hi, I just discovered AlDente following the Git discussions about the M1 processor (wich I have), I just installed your smart application and would leave you a little "Thank you" about sharing AlDente, so : Thanks a lot.

Have a nice day

Kevin

**Comment by MatthiasKerbl** [comment]
Hi Kevin,
we are glad you like it. :)

Have a nice day too,

Matthias
